server/mscalendar/daily_summary.go

```diff
@@ -92,6 +93,65 @@ func (m *mscalendar) SetDailySummaryEnabled(user *User, enable bool) (*store.Dai
 	return dsum, nil
 }
 
+func (m *mscalendar) processAllDailySummaryByUser(now time.Time, userIndex store.UserIndex) error {
```

Comment:
`ByUser` makes me think "for one user". Maybe `processAllDailySummaryWithIndividualCredentials`